modules/core/test/test_umat.cpp

```diff
@@ -1419,4 +1419,36 @@ TEST(UMat, resize_Mat_issue_13577)
     cv::ocl::setUseOpenCL(useOCL);  // restore state
 }
 
+TEST(UMat, exceptions_refcounts_issue_20594)
+{
+    if (!cv::ocl::useOpenCL())
+    {
+        // skip test, difficult to create exception scenario without OpenCL
+        std::cout << "OpenCL is not enabled. Skip test" << std::endl;
+        return;
+    }
+
+    UMat umat1(10, 10, CV_8UC1);
+    EXPECT_EQ(umat1.u->refcount, 0);
```

Comment:
`_EQ` checks in Google tests have this order of arguments: `(expected, actual)`.
So they should be swapped here and below.

Reply:
ok